numba/tests/test_parfors.py

```diff
@@ -877,6 +880,13 @@ def test_impl(A):
         self.assertTrue(countParfors(test_impl, (types.Array(types.float64, 1, 'C'), )) == 2)
         self.assertTrue(countParfors(test_impl, (types.Array(types.float64, 2, 'C'), )) == 2)
 
+    @skip_unsupported
+    def test_issue4963(self):
+        def test_impl():
+            buf = np.zeros((_GLOBAL_INT_FOR_TESTING1, _GLOBAL_INT_FOR_TESTING2))
+            return buf
+        self.check(test_impl)
+
```

Comment:
I think it'd be good to also add a freevar variant:
```python
    _FREEVAR_INT_FOR_TESTING1 = 17
    _FREEVAR_INT_FOR_TESTING2 = 5

    @skip_unsupported
    def test_issue4963_freevars(self):
        def test_impl():
            buf = np.zeros((_FREEVAR_INT_FOR_TESTING1, _FREEVAR_INT_FOR_TESTING2))
            return buf
        self.check(test_impl)
```

Reply:
Merged.

Reply:
Thanks!